docs/source/model/dependencies.rst

```diff
@@ -394,6 +394,100 @@ This way, MLflow will copy the entire ``src/`` directory under ``code/`` and you
 
     By the same reason, the ``code_paths`` option doesn't handle the relative import of ``code_paths=["../src"]``.
 
+Limitation of ``code_paths`` in loading multiple models with the same module name but different implementations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The current implementation of the ``code_paths`` option has a limitation that it doesn't support reloading the model with the same module name but different implementations. The following example illustrates this limitation.
```

Comment:
```suggestion
The current implementation of the ``code_paths`` option has a limitation that it doesn't support simultaneously loading multiple models that depends on modules with the same name but different implementations. The following example illustrates this limitation.
```
nit: per my understanding, this might be more accurate description for the situation?

Reply:
Good point. More accurate way to describe this is `loading multiple models ... within the same python session`. `simultaneously` might make users wonder "what if they are loaded one by one"?

Reply:
Makes sense!